systers_portal/templates/account/email.html

```diff
@@ -0,0 +1,63 @@
+
+{% extends "base.html" %}
+
+{% load account %}
+
+{% block title %} - Email{% endblock %}
+
+{% load crispy_forms_tags %}
+
+{% block content %}
+
+<h1>E-mail Addresses</h1>
```

Comment:
Please add a `<hr>` after the heading to conform to the styleguide of the whole website.